src/style.css

```diff
@@ -1,3 +1,159 @@
 body {
   margin: 0;
+  --font-family: "Roboto", sans-serif;
+  --active-star: url("./images/star-active.svg");
+  --empty-star: url("./images/star.svg");
+  --button-color: #00acdc;
+  --main-color: #060b35;
 }
+
+h2,
+h3,
+h4,
+p {
+  margin: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--0 .stars__star:nth-child(-n) {
+  background-image: url(./images/star-active.svg);
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  min-height: 406px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  display: inline-block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+  font-family: var(--font-family);
+}
+
+.card__code {
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__stars {
+  width: 96px;
+  height: 16px;
+}
+
+.card__reviews {
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  margin: 0;
+
+  color: #616070;
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-family: var(--font-family);
```

Comment:
if you need same font-family for all or most of your components you can set it on parent component like body